test/system/helpers.bash

```diff
@@ -55,6 +55,18 @@ function basic_setup() {
     PODMAN_TMPDIR=$(mktemp -d --tmpdir=${BATS_TMPDIR:-/tmp} podman_bats.XXXXXX)
 }
 
+# Check that an unprivileged user cannot read or write to the storage
+function check_no_unprivileged_access() {
+    GRAPH_ROOT=$(run_podman info --format json | jq .store.GraphRoot)
+    RUN_ROOT=$(run_podman info --format json | jq .store.RunRoot)
+    find $GRAPH_ROOT $RUN_ROOT -perm -o+w | while read i; do
+        if chroot --userspec 1000:1000 / bash -c 'chmod +w $(dirname $i); chmod +w $i; cat $i || echo hello > $i || echo hello > $i/test' 2> /dev/null; then
```

Comment:
Careful with plain `$i`! Any space in the filename or any parent directory will cause all of these to fail. The convention is to double-quote every reference: `"$i"`. (It's even better to use `find ... -print0 | xargs -0`, but that would be harder to read in this case.

Reply:
thanks, fixed now